t/errlog.t

```diff
@@ -932,3 +932,66 @@ maybe log lines: #1
 end
 \z
 --- skip_nginx: 2: <1.11.2
+
+
+
+=== TEST 23: the system default filtering level is "debug"
+--- config
+    location /t {
+        content_by_lua_block {
+            local errlog = require "ngx.errlog"
+            ngx.print('Is "debug" the system default filtering level? ',
+                      errlog.get_sys_filtering_level() == ngx.DEBUG)
+        }
+    }
+--- log_level: debug
+--- request
+GET /t
+--- response_body chomp
+Is "debug" the system default filtering level? true
+
+
+
+=== TEST 24: the system default filtering level is "emerg"
+--- config
+    location /t {
+        content_by_lua_block {
+            local errlog = require "ngx.errlog"
+            ngx.print('Is "emerg" the system default filtering level? ',
+                      errlog.get_sys_filtering_level() == ngx.EMERG)
+        }
+    }
+--- log_level: emerg
+--- request
+GET /t
+--- response_body chomp
+Is "emerg" the system default filtering level? true
+
+
+
+=== TEST 25: get system default filtering level during Nginx worker starts
+--- http_config
+    init_worker_by_lua_block {
+        local errlog = require "ngx.errlog"
+        package.loaded.log_level = errlog.get_sys_filtering_level()
+    }
+--- config
+    location /t {
+        content_by_lua_block {
+            local log_level = package.loaded.log_level
+
+            if log_level >= ngx.WARN then
+                ngx.log(ngx.WARN, "log a warning event")
+            end
+            if log_level >= ngx.NOTICE then
+                ngx.log(ngx.NOTICE, "log a notice event")
```

Comment:
Because this is logged with `ngx.NOTICE`, it will never get logged anyways, even if the surrounding condition fails. I think it should rather be:
```lua
if log_level >= ngx.NOTICE then
    ngx.log(ngx.WARN, "do not log another warning event")
end
```

This way, we make sure that the logging was not written due to the surrounding condition, and not due to the `ngx.log` logging level.